tests/support/asyncio_utils.py

```diff
@@ -0,0 +1,40 @@
+# pylint: disable=locally-disabled, missing-class-docstring, missing-function-docstring, redefined-outer-name, too-few-public-methods, missing-module-docstring
+
+import asyncio
+
+from jellyfish import Notifier
+
+
+async def assert_events(notifier: Notifier, event_checks: list):
+    await _assert_messages(notifier.on_server_notification, event_checks, 1.)
+
+
+async def assert_metrics(notifier: Notifier, metrics_checks: list):
+    await _assert_messages(notifier.on_metrics, metrics_checks, 2.)
```

Comment:
Do these timeouts need to be different? Can we unify them under the same constant, maybe?